modules/calib3d/src/usac.hpp

```diff
@@ -801,7 +801,8 @@ bool solvePnPRansac( InputArray objectPoints, InputArray imagePoints,
 Mat findEssentialMat( InputArray points1, InputArray points2,
                       InputArray cameraMatrix1,
                       int method, double prob,
-                      double threshold, OutputArray mask);
+                      double threshold, OutputArray mask,
+                      int maxIters=1000);
```

Comment:
It's internal function, not public API. Default value looks a bit suspicious. Please check if it's used in other places. Most probably `maxIters` should be promoted there too.